packages/components/molecules/f-restaurant-card/src/components/RestaurantCard.vue

```diff
@@ -474,11 +518,28 @@ export default {
             order: initial;
         }
     }
+
+    .c-restaurantCard-ratingSkeleton {
+        .c-restaurantCard--listItem .c-restaurantCard-content & {
+            grid-column: 1;
+            width: 100%;
+        }
+    }
+
+    .c-restaurantCard--hideOnMobile {
+        .c-restaurantCard--listItem .c-restaurantCard-content & {
+            display: block;
+        }
+    }
```

Comment:
There's a lot of duplication of `.c-restaurantCard--listItem .c-restaurantCard-content &` in the file. I wonder if we can do something like this in such cases:

```suggestion
    
   .c-restaurantCard--listItem .c-restaurantCard-content {
        .c-restaurantCard-ratingSkeleton {
            grid-column: 1;
            width: 100%;
        }
        
        .c-restaurantCard--hideOnMobile {
            display: block;
        }
    }
    
```

Reply:
great suggestion - much tider - thank you!

Reply:
I think we should minimise the use of 
```
.child {
        .parent & {
            ...
        }
    }
```
unless we need it, it improves readability or reduces duplication.